index.rst

```diff
@@ -1,29 +1,919 @@
+The report concludes with a section on findings that are pertinent to the commissioning team but fall outside the scope of the original `charge`_ .
+It is recommended that these issues get addressed by either a follow-on charge or another mechanism.
+
+Rapid Analysis Capability
+-------------------------
+
+Most of the envisioned FAFF functionality requires data products based on both observatory events and/or telemetry as well as camera images to be available for immediate use at the summit in order to display this information and thereby inform nighttime operations.
+While developing use cases, we recorded the applicable input data products and associated timescales for their use, and then compiled these into a consolidated list to better understand the set of required data processing, analysis, artifact creation and handling mechanisms.
+
+We find that multiple FAFF use cases require a `Rapid Analysis <https://confluence.lsstcorp.org/display/LSSTCOM/Rapid+Analysis+Use-Case>`_ capability that includes automated SFP of camera images through instrument signature removal and source detection on the 30-60 second timescale to enable the creation of various data quality metrics and data visualizations.
+The Confluence page linked above describes the needed data products and timescales for their creation, as requested in `charge`_  question 2.
+The use cases developed for charge question 1 assume that the data products from this initial processing step are available.
+
+It should be noted that Rapid Analysis appears to share many of the same functionalities as what is often referred to as Prompt Processing.
+In writing this report, it was identified that the Prompt Processing terminology is used to refer to slightly different things depending on author and or person, resulting in significant confusion.
+In this report, we refer to `Prompt Processing`_ as the *framework* for continuously processing the stream of images coming off the telescope.
+This framework is used to execute a payload, such as the Science Pipeline payloads for Alert Production including Solar System Processing.
+Prompt Processing is to be run at the United States Data Facility (USDF).
+
+The needs for near-realtime data quality assessment by the commissioning team could be partially addressed by a suitable payload executed with the Prompt Processing framework at USDF, provided that results are made available to the summit.
+The payload would be the pared-down SFP, augmented with the calculation of metrics and/or values that are pertinent for observatory diagnostics, which then requires a mechanism to report those metrics back into the control system framework such that potential alerts signaling abnormal data can be generated for observers.
+Throughout this report, this summit-dedicated payload is referred to as Rapid Analysis, whose instantiation would ensure that on-the-fly diagnostic information is available to observers.
```

Comment:
Maybe add "payload" after "Rapid Analysis" and emphasize it with quotes, e.g.;

> Throughout this report, this summit-dedicated payload is referred to as "Rapid Analysis payload", whose instantiation would ensure that on-the-fly diagnostic information is available to observers.

Reply:
I think that's grammatically incorrect... and uses payload twice in the same sentence.
How about:

> Throughout this report, this summit-dedicated data processing is referred to as the "Rapid Analysis payload" or sometimes just "Rapid Analysis," whose instantiation would ensure that on-the-fly diagnostic information is available to observers.

Reply:
Sounds good